setup.cfg

```diff
@@ -0,0 +1,75 @@
+[metadata]
+name = amsterdam-schema-tools
+version = 0.18.1
+url = https://github.com/amsterdam/schema-tools
+license = Mozilla Public 2.0
+author = Amsterdam Data en Informatie
+author_email = datapunt@amsterdam.nl
+description = Tools to work with Amsterdam schema.
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+[options]
+packages = find:
+python_requires = >= 3.7
+install_requires =
+    geoalchemy2
+    psycopg2
+    pg-grant
+    click
+    deepdiff
+    jsonschema
+    ndjson>=0.3.0
+    shapely
+    python-string-utils
+    python-dateutil
+    requests
+    jinja2
```

Comment:
NB Chris has added some pinning here (security-related) in the `old` setup.py

Reply:
I'll look them up and will pin them here as well.

Reply:
I've have looked at [setup.py](https://github.com/Amsterdam/schema-tools/blob/master/setup.py) on `master` but I see no additional pinned versions. Are you referring to another project?

Reply:
Ah, this one https://github.com/Amsterdam/dso-api/pull/234

Will do the same here.

Reply:
On second thought, after having discussed it with @vdboor , I'll  leave the exact version pinning to the consumers of `schematools`. Eg, the applications that actually use this library. I will add `Django >= 3.0`, though. To state the minimum requirement.